aten/src/ATen/native/nested/NestedTensorTransformerFunctions.cpp

```diff
@@ -8,6 +8,32 @@
 namespace at {
 namespace native {
 
+Tensor NestedTensor_linear(
```

Comment:
This essentially rewrites the matmul func defined below but with at::linear instead of at::matmul, don't love that

Reply:
Yup, could be interesting to see some of that logic factored out